draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -763,29 +1020,63 @@ As a rough style guide the following convention is used:
 
 - c1, c2, ...: are constant values, which can be computed in advance.
 
-## Clearing Cofactor
+## Clearing the cofactor {#cofactor-clearing}
```

Comment:
That's a good point.

On the other hand, "clearing the cofactor" is essentially standard terminology. It might be confusing to make up alternate terminology.

Reply:
We can stay with clearing the cofactor and making the observation that clearing the cofactor for pairing curves is different from a straightforward multiplication by h.

Reply:
Great, this makes sense.

I've pushed a commit that I believe addresses this comment:

1. it clarifies the three cases that @armfazh lists above,
2. it defines a new abstract function `clear_h(x, y)` that represents cofactor clearing, and
3. it replaces `Output h * (x, y)` with `clear_h(x, y)` in the descriptions of the implementations.

Reply:
Note that after #111 these `clear_h()` operations should be revisited.